### PR TITLE
[plugin-web-app] Add step to save CSS property value to variable

### PR DESCRIPTION
**Pull request by draker94**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 1]
```diff
@@ -1382,6 +1382,27 @@ When I save number of open tabs to $scopes variable `$variableName`
...
+
+Saves the CSS property value of the element located by locator into a variable.
+
+[source,gherkin]
+----
+When I save `$cssProperty` CSS property value of element located by `$locator` to $scopes variable `$variableName`
+----
+
+* `$cssProperty` - The name of an https://www.w3schools.com/cssref/index.php[CSS property].
+* `$locator` - <<_locator>>.
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:1395]:
```suggestion
* `$locator` - The <<_locator,locator>> used to find the element to get the CSS property.
```

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 2]
```diff
@@ -278,6 +281,39 @@ public void saveTableToContext(Set<VariableScope> scopes, String variableName)
...
+     *                       <i>Available scopes:</i>
+     *                       <ul>
+     *                       <li><b>STEP</b> - the variable will be available only within the step,
+     *                       <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     *                       <li><b>STORY</b> - the variable will be available within the whole story,
+     *                       <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     *                       </ul>
+     * @param variableName   The name of the variable to save the CSS property value
+     */
+    @When("I save `$cssProperty` CSS property value of element located by `$locator` to $scopes"
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:305]: please move it to `ElementSteps` class

File: vividus-plugin-web-app/src/test/java/org/vividus/steps/ui/web/SetVariableStepsTests.java [hunk 3]
```diff
@@ -253,4 +257,29 @@ void shouldCallJSScriptAndSaveValueToContext()
         setVariableSteps.saveTableToContext(VARIABLE_SCOPE, VARIABLE_NAME);
         verify(variableContext).putVariable(VARIABLE_SCOPE, VARIABLE_NAME, listOfTables);
     }
+
+    @Test
+    void shouldSaveCssPropertyValue()
+    {
+        Locator locator = mock(Locator.class);
+        WebElement webElement = mock(WebElement.class);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/test/java/org/vividus/steps/ui/web/SetVariableStepsTests.java:265]:
```suggestion
        Locator locator = mock();
        WebElement webElement = mock();
```

File: vividus-plugin-web-app/src/test/java/org/vividus/steps/ui/web/SetVariableStepsTests.java [hunk 4]
```diff
@@ -253,4 +257,29 @@ void shouldCallJSScriptAndSaveValueToContext()
...
+
+        setVariableSteps.saveCssPropertyValue(CSS_PROPERTY, locator, VARIABLE_SCOPE, VARIABLE_NAME);
+        verify(variableContext).putVariable(VARIABLE_SCOPE, VARIABLE_NAME, VALUE);
+        verifyNoInteractions(softAssert);
+    }
+
+    @Test
+    void shouldNotSaveCssPropertyValueIfTheValueIsNotFound()
+    {
+        Locator locator = mock(Locator.class);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/test/java/org/vividus/steps/ui/web/SetVariableStepsTests.java:278]:
```suggestion
        Locator locator = mock();
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java [hunk 5]
```diff
@@ -278,6 +281,39 @@ public void saveTableToContext(Set<VariableScope> scopes, String variableName)
...
+            + " variable `$variableName`")
+    public void saveCssPropertyValue(String cssProperty, Locator locator, Set<VariableScope> scopes,
+            String variableName)
+    {
+        baseValidations.assertElementExists("The element to get the CSS property value", locator)
+                .map(e -> webElementActions.getCssValue(e, cssProperty))
+                .ifPresentOrElse(value -> saveVariable(scopes, variableName, value),
+                        () -> softAssert.recordFailedAssertion(
+                                String.format("The '%s' CSS property does not exist", cssProperty)));
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java:314]: it looks like this message will be also logged if element is not found, but it should be logged only if css prop doesn't exist

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**